CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 # go-ipfs changelog
 
-### 0.4.10 - 2017-06-18
+### 0.4.10 - 2017-06-27
+
+Ipfs 0.4.10 is a patch release that contains several exciting new features,
+bugfixes and general improvements.
+
+The `ipfs pin` command got two new subcommands, `verify` and `update`. `ipfs
+pin verify` is used to scan the repo for pinned object graphs and check their
+integrity. Any issues are reported back in a way that should make it relatively
+easy to recover from. This subcommand was added to help recover from datastore
```

Comment:
"reported back in a way that should make it relatively easy to recover from."
--> what ways do we mean? ;)
Maybe more specific like "reported back with helpful error text"